README.md

```diff
@@ -53,124 +50,38 @@ import tidy3d.web as web
-
-The easiest way to install the tidy3d python interface is through [pip](https://pypi.org/project/tidy3d/).
-
-```
-pip install tidy3d
-```
-
-This will install the latest stable version.
-
-To get a specific version `x.y.z`, including the "pre-release" versions, you may specify the version in the command as:
-
-```
-pip install tidy3d==x.y.z
-```
-
-### Installing from source
-
-Alternatively, for development purposes, eg. developing your own features, you may download and install the package from source as:
-
-```
-git clone https://github.com/flexcompute/tidy3d.git
-cd tidy3d
-pip install -e .
-```
-
-### Configuring and authentication
-
-With the front end installed, it must now be configured with your account information, which is done via an "API key".
-
-You can find your API key in the [web interface](http://tidy3d.simulation.cloud). After signing in and navigating to the account page by clicking the "Account Center" icon on the left-hand side. Then, click on the "API key" tab on the right hand side of the menu and copy your API key.
-
-Note: We refer to your API specific API key value as `XXX` below.
-
-To link your API key with Tidy3D, you may use one of three following options:
-
-#### Command line (recommended)
-
-The easiest way is through the command line via the `tidy3d configure` command. Run:
-
-```
-tidy3d configure
-```
-
-and then enter your API key `XXX` when prompted.
-
-Note that Windows users must run the following instead (ideally in an anaconda prompt):
-
-```
-pip install pipx
-pipx run tidy3d configure
-```
-
-You can also specify your API key directly as an option to this command using the `api-key` argument, for example:
-
-```
-tidy3d configure --apikey=XXX
-```
-
-#### Manually
-
-Alternatively, you can manually set up the config file where Tidy3D looks for the API key. The API key must be in a file called `.tidy3d/config` located in your home directory, containing the following
+To test the authentication, you may try importing the web interface via.
 
 ```
-apikey = "XXX"
+python -c "import tidy3d.web"
 ```
 
-You can manually set up your file like this, or do it through the command line line:
-
-``echo 'apikey = "XXX"' > ~/.tidy3d/config``
-
-Note the quotes around `XXX`.
-
-Note that Windows users will most likely need to place the `.tidy3d/config` file in their `C:\Users\username\` directory (where `username` is your username).
-
-#### Environment Variable
-
-Lastly, you may set the API key as an environment variable named `SIMCLOUD_APIKEY`.
-
-This can be set up using
-
-``export SIMCLOUD_APIKEY="XXX"``
-
-Note the quotes around `XXX`.
-
-### Testing the installation and authentication
-
-#### Front end package
+It should pass without any errors if the API key is set up correctly.
 
-You can verify the front end installation worked by running:
+To get started, our documentation has a lot of [examples](https://docs.flexcompute.com/projects/tidy3d/en/latest/notebooks/index.html) for inspiration.
```

Comment:
this link was broken when I tried it.

Reply:
So yeah, they're currently broken. It's a bit tricky really, because in a future pre/2.6 will be depreciated, and then we'll need to do another PR before the release. So was just thinking maybe get it right to what it'll be in the future when it's fully released?

Reply:
ok that's fine.